packages/shared/lib/services/plans/definitions.ts

```diff
@@ -1,237 +1,137 @@
+import { enterprisePlanFlags, freePlanFlags, growthPlanFlags, starterPlanFlags } from './flags.js';
+
 import type { PlanDefinition } from '@nangohq/types';
 
-export const freePlan: PlanDefinition = {
-    code: 'free',
+/**
+ * Latest plans
+ */
+
+export const freeV2Plan: PlanDefinition = {
+    name: 'free',
+    orbVersion: 2,
     title: 'Free',
     description: 'For hobby and testing.',
+    isPaid: false,
     prevPlan: null,
-    nextPlan: ['starter', 'growth', 'enterprise'],
-    orbId: 'free',
+    nextPlan: null, // Will be set after all plans are defined
     canChange: true,
     basePrice: 0,
-    flags: {
-        api_rate_limit_size: 'm',
-        environments_max: 2,
-        has_otel: false,
-        has_sync_variants: false,
-        connections_max: 10,
-        records_max: 100_000,
-        proxy_max: 100_000,
-        function_executions_max: 100_000,
-        function_compute_gbms_max: 50_000_000,
-        webhook_forwards_max: 100_000,
-        function_logs_max: 100_000,
-        name: 'free',
-        sync_frequency_secs_min: 3600,
-        auto_idle: true,
-        monthly_actions_max: 1000,
-        monthly_active_records_max: 5000,
-        has_webhooks_script: false,
-        has_webhooks_forward: false,
-        can_override_docs_connect_url: false,
-        can_customize_connect_ui_theme: false,
-        can_disable_connect_ui_watermark: false
-    },
-    display: {
-        features: [
-            { title: '10 connections' },
-            { title: '1k actions' },
-            { title: '5k synced records' },
-            { title: '2 environments' },
-            { title: 'API authorization' },
-            { title: 'Syncs & actions' },
-            { title: 'MCP & AI Tools' },
-            { title: 'Proxy requests' }
-        ],
-        sub: 'Certain features of free projects (syncs & actions) are paused after two weeks of inactivity.'
-    }
+    flags: freePlanFlags
```

Comment:
instead of creating a flags file you could set them in one planDefinition and reference it in the other one `flags: freeV2Plan.flags`. It keeps everything in one file but remove the duplication